design-documents/asynchronous-import/base-extension.md

```diff
@@ -68,30 +130,16 @@ In this case input request will looks like:
       "pieces_count": "5"
       "piece_number": "1",
       "import_type": "base64_encoded_data",
-      "source_type": "csv",
-      "uuid": "UUID" // Any UUID transferred by request creator. Not Required field
+      "uuid": "UUID"
```

Comment:
Why is `format` field not required here?

Reply:
Yes you are right, forgot it.
But here is a reasonable question, if we are transferring multiple parts,  basically we need to transfer format only 1 time, which is ok.
But what IF customer transfer it in each body and some paramether will be different in some part? 
I would propose always use values from first request. If you want to change them, you have to use PUT

Reply:
I would prefer to avoid any intelligence in such a scenario. Definition what is "first" request not clear as a client may initiate requests with all parts simultaneously. If requests have conflicting or missed data - throw an error and ask a client to resolve it.

As we don't have any instrument to declare "property required among several requests" I would prefer to make it required for requests with all parts. As this API created to process big amount of data, duplication of fields in request will not add significant overhead.

Reply:
Agree with @vkublytskyi there is no difference for a client app wether to include all data with each request or not. However it simplifies server side implementation by eliminating unnecessary state.